lenovo/yoga/6/13ALC6/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, pkgs, ...  }:
+
+{
+  boot.initrd.kernelModules = [ 
+    "amdgpu"
+    "ideapad_laptop"
+  ];
+  services.xserver.videoDrivers = [ "amdgpu" ];
```

Comment:
It would be nice to re-use some amd gpu hardware module rather than copy and pasting all those.